pandas/core/frame.py

```diff
@@ -6177,14 +6177,13 @@ def stack(self, level=-1, dropna=True):
 
     def explode(self, column: Union[str, Tuple]) -> "DataFrame":
         """
-        Transform each element of a list-like to a row, replicating the
-        index values.
+        Transform each element of a list-like to a row, replicating index values.
 
         .. versionadded:: 0.25.0
 
         Parameters
         ----------
-        column : str or tuple
+        column : str or tuple column name to explode
```

Comment:
Can you add the description to the next line, in the same line of the parameter name we should have the expected type, and the default value if it exists.